apps/website-25/src/lib/api/db/tables.ts

```diff
@@ -28,26 +28,32 @@ export const courseTable: Table<Course> = {
 
 export interface Unit extends Item {
   id: string,
-  courseId: string,
+  courseTitle: string,
   title: string,
   content: string,
+  unitNumber: string,
   duration: number,
+  description: string,
 }
 
 export const unitTable: Table<Unit> = {
   name: 'Unit',
   baseId: 'appbiNKDcn1sGPGOG',
   tableId: 'tblsDKJ8VCyO619nk',
   mappings: {
-    courseId: 'fldLmQZ0ISTr7xQUE',
```

Comment:
I think you probably do want to keep courseId here - I suspect it's quite useful for lots of other linking?